wolfssl/tests/ping_pong.rs

```diff
@@ -192,18 +193,27 @@ async fn server<S: SockIO>(sock: S, protocol: Protocol) {
     println!("[Server] Finished");
 }
 
-#[tokio::test]
-async fn dtls() {
-    use Protocol::*;
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_3 => panics; "client_1.2_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2; "client_1.2_server_1.2")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServer; "client_1.2_server")]
```

Comment:
WDYT of including `any` in place of the version here to clarify and make the pattern consistently `(client|server)_($version|any)`?

Reply:
Yep, sounds good